### PR TITLE
(New) Another Timeline Test [pr]

### PR DESCRIPTION
**Pull request by pdb493**
this is a new description

### CODE REVIEW

File: app/User.php [hunk 1]
```diff
@@ -1,6 +1,5 @@
 <?php
 namespace App;
-
```
[the remark this answers is not in the document]
Comment (pdb493) [reply app/User.php:3]: 3. solo reply

File: app/User.php [hunk 2]
```diff
@@ -23,13 +22,7 @@ class User extends Authenticatable
      * @var array
      */
     protected $hidden = [
-        'password', 'remember_token',
+        'password', 
+	'remember_token',
```
[the remark this answers is not in the document]
Comment (pdb493) [reply app/User.php:26]: 4. reply with summary

**Review by pdb493 — COMMENTED** [review]
4. summary with reply

File: app/User.php (hunk 1 above)
[the remark this answers is not in the document]
Comment (pbarsallo) [reply app/User.php:3]: 5. reply

File: config/app.php [hunk 3]
```diff
@@ -2,6 +2,9 @@
 
 return [
 
+
+// TEST COMMENT
```
Comment (pbarsallo) [inline config/app.php:6]: 5. inline

**Review by pbarsallo — APPROVED** [review]
5. approval summary

**Comment by pdb493** [comment]
> 1. approval summary

6. quote reply

**Review by pbarsallo — CHANGES_REQUESTED** [review]
7. request changes

**Comment by pdb493** [comment]
asdf

**Comment by pdb493** [comment]
> asdf

yes

**Pull request closed without merging**